hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/kafka/SqlAvroTest.java

```diff
@@ -793,7 +794,7 @@ public void test_topLevelFieldExtraction() {
 
         insertRecord(1, "Alice");
 
-        assertRowsEventuallyInAnyOrder(
+        assertRowsEventuallyInAnyOrder(client(),
```

Comment:
why only this test executed with `client`? It also not executed using member API

Reply:
I am trying to trigger serialization of some Avro records, and the most _convincing_ way of doing that is to read them via a client proxy. Inter-member communication may also need to be tested, which can be easily done by executing the `SELECT` statement `file.SqlAvroTest.test_complexAvroTypes()` for the non-master node as well.

> It also not executed using member API

Do you mean `HazelcastInstance.getSql().execute()`?

Reply:
> Do you mean `HazelcastInstance.getSql().execute()`?

I meant `assertRowsEventuallyInAnyOrder(instance(),`

Reply:
> I am trying to trigger serialization of some Avro records, and the most convincing way of doing that is to read them via a client proxy.

If we want to have 1 test checking client serialization of Avro, then it should be a dedicated test, not "accidental" use in an apparently randomly chosen test.

Reply:
1. `file.SqlAvroTest.test_complexAvroTypes()` tests all complex Avro types via SQL, and
2. `AvroSourceTest.testSpecificReader()` tests `SpecificRecord`s via Jet.

Both tests trigger inter-member and member-to-client serialization. Do you want a dedicated test for serialization?

Reply:
it is sufficient